applications/nrf5340_audio/src/modules/lc3_streamer.h

```diff
@@ -12,14 +12,20 @@
 #include <stdbool.h>
 #include <sys/types.h>
 
+struct lc3_stream_cfg {
```

Comment:
Discuss if we should keep as is, rename, or just use these three variables direct to `lc3_streamer_file_header_verify`